docs/system/wind_turbine.md

```diff
@@ -1,7 +1,147 @@
+
+| Advantages                            | Drawbacks                                     |
+|---------------------------------------|-----------------------------------------------|
+| High Power coefficient                | Yaw system to follow wind needed              |
+| No pulsating torque on drive train    | High noise generation due to high tip speed   |
+| Well understood and developed         | Complex blade design and production           |
+
+#### VAWT
+
+Vertical axis wind turbines have a power coefficient of $c_p \approx 30\%-40\%$ and are not often found in commercial setups. They have however some advantages that makes them suitable for a range of applications. Namely their low noise profile and unusal appearance give them high acceptance in urban areas. They also do not require a yaw system and are better suited for turbulent areas. Most common designs are the Darriues turbine, the helical bladed turbine and the straight-bladed Darrieus turbines (see [Wikipedia](https://en.wikipedia.org/wiki/Darrieus_wind_turbine) for more details)
+
+| Advantages                            | Drawbacks                                     |
+|---------------------------------------|-----------------------------------------------|
+| No yaw mechanism needed               | Medium Power coefficient                      |
+| Low noise profile                     | High normal forces on axis                    |
+| Simple blade geometry                 | Oscillating torque on generator               |
+
+## Generators
+
+Electric generators can be classified into radial flux machines and axial flux machines. The main difference is the flow direction of magnetic flux, where it is either parallel or radial to the rotaional axis. The common synchronous and asynchronous (induction) machines belong to the radial flux machines and are mainly used in large turbines. One of the main challenges for larger wind turbines is the connection to the grid. While DC-Sources can be connected to the grid when their voltage is equal to the grid voltage, AC-Sources have to be in synch with the grid's voltage frequency and amplitude [4]. For small scale, especially off-grid turbines, axial flux generators are more common as they can be fairly easily constructed without high level machinery, especially for low-rpm usecases.
+
+### Asynchronous
+
+The distinctive property of induction generators is the slip between rotor and stator fields. In generator operation, the turbine moves the rotor *above* the synchronous speed. Because of the two magnetic fields running against each other, this type of generator can introduce high reactive power loads in the system. 
```

Comment:
An asynchronous machine always needs an excitation of the stator field, correct? So this could also be mentioned as a distinctive property, as it makes it much more difficult to use in "simple" applications because you need additional sophisticated power electronics and not only a simple diode bridge rectifier to charge a battery, for example.